sopel/tools/jobs.py

```diff
@@ -33,7 +33,14 @@ def __exit__(self, _type, _value, _traceback):
 
 
 class PriorityQueue(Queue.PriorityQueue):
-    """A priority queue with a peek method."""
+    """A priority queue with a peek method.
+
+    .. deprecated:: 7.0
+
+        This will be removed in Sopel 8. Use Python's built-in priority queue
```

Comment:
Can this "priority queue" link directly to the Python docs for that type (I guess you refer to [`queue.PriorityQueue`](https://docs.python.org/3/library/queue.html#queue.PriorityQueue))?

Reply:
It can, and now it does.